Source/AssetRipper.Processing/Editor/EditorFormatProcessor.cs

```diff
@@ -79,8 +79,19 @@ public void Process(GameData gameData)
 			Convert(asset);
 		}
 
-		//Parallel processing
-		Parallel.ForEach(GetReleaseAssets(gameData), ConvertAsync);
```

Comment:
If this changes the output, it's a bug. ConvertAsync should only operate on a single asset and not reference any others.

Reply:
The order of calls to the global RNG would be susceptible to variations in execution timing if it ran in parallel.